### PR TITLE
Add option for custom logos on the sign in page

### PR DESCRIPTION
**Pull request by JoelSpeed**
## Description

This adds a logo to the sign in page and the option for users to disable it or provide their own logo if they want to.
Horizontal

## Motivation and Context

Gives users the ability to make their sign in page more custom to their own uses.

## How Has This Been Tested?

Manually using the local testing environment.

## Checklist:

- [x] My change requires a change to the documentation or CHANGELOG.
- [x] I have updated the documentation/CHANGELOG accordingly.
- [x] I have created a feature (non-master) branch for my PR.

### CODE REVIEW

File: docs/docs/configuration/overview.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ An example [oauth2-proxy.cfg](https://github.com/oauth2-proxy/oauth2-proxy/blob/
 | `--cookie-secure` | bool | set [secure (HTTPS only) cookie flag](https://owasp.org/www-community/controls/SecureFlag) | true |
 | `--cookie-samesite` | string | set SameSite cookie attribute (`"lax"`, `"strict"`, `"none"`, or `""`). | `""` |
 | `--custom-templates-dir` | string | path to custom html templates | |
+| `--custom-sign-in-logo` | string | path to an custom image for the sign_in page logo. Use \"-\" to disable default logo. |
```
Comment (NickMeves) [inline docs/docs/configuration/overview.md:43]: Did we only want to support local paths for the logo image? Or also allow a direct link to where it is already hosted (e.g. CDN)?

**Review by NickMeves — APPROVED** [review]
LGTM! My 1 question isn't blocking, feel free to merge.

File: docs/docs/configuration/overview.md (hunk 1 above)
Comment (JoelSpeed) [reply docs/docs/configuration/overview.md:43]: That's a good idea, I hadn't really thought about it. My suggestion would be to wait to see if someone wants this, or if one of us decides it's interesting enough to look into at a future date. I suspect once we get a new release and people see there's now the logo option, we they will let us know if they'd prefer HTTP links to the image

Comment (Raboo) [reply docs/docs/configuration/overview.md:43]: I think also allowing a direct link (CDN) would be a smart idea...

**Pull request merged**